test/run-spec-wasm2c.py

```diff
@@ -173,6 +187,31 @@ def _CacheModulePrefixes(self):
                 self.module_prefix_map[name_idx] = name
                 self.unmangled_names[name_idx] = command['as']
 
+    def _WriteModuleInstanceImports(self, command, uninstantiable):
+        header_filename = utils.ChangeExt(command['filename'], '.h')
+        with open(os.path.join(self.out_dir, header_filename), encoding='utf-8') as f:
+            imported_modules = set()
+            for line in f:
+                if "import: " in line:
+                    line_split = line.split()
+                    import_module_name = MangleName(line_split[2][1:-1])
+                    imported_modules.add(import_module_name)
+
+            if (uninstantiable):
+                self.out_file.write("ASSERT_TRAP(")
+
+            if len(imported_modules) > 0:
+                self.out_file.write("%s_init(&%s_module_instance" % (self.GetModulePrefix(), self.GetModulePrefix()))
+                for imported_module in sorted(imported_modules):
+                    self.out_file.write(", &%s_module_instance" % imported_module)
+                self.out_file.write(")")
```

Comment:
Won't this code work fine even for the case where imported_modules is empty?  i.e. do you need the if/else here?

Reply:
Good catch -- change made.